client/src/views/dashboard.js

```diff
@@ -0,0 +1,77 @@
+import Snabbdom from 'snabbdom-pragma'
+import layout from './layout'
+import { formatTime, formatSat, formatNumber } from './util'
+
+const staticRoot = process.env.STATIC_ROOT || ''
+const isTouch = process.browser && ('ontouchstart' in window)
+
+const homeLayout = (body, { t, activeTab, ...S }) => layout(
+ <div>
+    { body }
+  </div>
+, { t, isTouch, activeTab, ...S })
+
+export const dashBoard = ({ t, blocks, dashboardState, loading, ...S }) => {
+  const { dashblocks, mempoolRecent } = dashboardState || {}
+
+return (homeLayout(
+  <div className="container dash-container">
+    <div className="block-container">
+      <div className="blocks-table">
+        <h3 className="table-title">{t`Latest Blocks`}</h3>
+        <div className="blocks-table-row header">
+          <div className="blocks-table-cell">{t`Height`}</div>
+          <div className="blocks-table-cell">{process.browser ? t`Timestamp` : t`Timestamp (UTC)`}</div>
+          <div className="blocks-table-cell">{t`Transactions`}</div>
+          <div className="blocks-table-cell">{t`Size (KB)`}</div>
+          <div className="blocks-table-cell">{t`Weight (KWU)`}</div>
+        </div>
+        { dashblocks && dashblocks.map(b =>
```

Comment:
Shouldn't this show the loading indicator when the block list is loaded, similarly to the mempool transactions below?